content-repo/extra-docs/articles/CI-CD-Article.md

```diff
@@ -0,0 +1,436 @@
+        ├── build_releated_scripts
+        │   ├── bucket_upload.py
+        │   ├── get_modified_packs.py
+        ├── dev_envs/pytest
+        │   ├── conftest.py
+        ├── Packs
+        │   ├── Pack1
+        │   │   ├── IncidentFields
+        │   │   │   ├── (your-incident-field.json)
+        │   │   │   ├── …
+        │   │   ├── IncidentTypes
+        │   │   │   ├── (your-incident-type.json)
+        │   │   │   ├── …
+        │   │   ├── Layouts
+        │   │   │   ├── (your-layout.json)
+        │   │   │   ├── …
+        │   │   ├── Playbooks
+        │   │   │   ├── (your-playbook.yml)
+        │   │   │   ├── …
+        │   │   ├── Scripts
+        │   │   │   ├── (your-script)
+        │   │   │   │   ├── (your-script.py)
+        │   │   │   │   ├── (your-script.yml)
+        │   │   │   ├── …
+        │   │   ├── Integrations
+        │   │   │   ├── (your-integration)
+        │   │   │   │   ├── (your-integration.py)
+        │   │   │   │   ├── (your-integration.yml)
+        │   │   │   ├── …
+        │   │   ├── ReleaseNotes
+        │   │   │   ├── (1_0_1.md)
+        │   │   │   ├── (1_0_2.md)
+        │   │   │   ├── …
+        │   ├── …
+        ├── .demisto-sdk-conf
+        ├── .gitignore
+        ├── .private-repo-settings
+        ├── CommonServerPowerShell.ps1
+        ├── CommonServerPython.py 
+        ├── demistomock.py
+        ├── demistomock.ps1  
+        ├── README.md
+        ├── requirements.txt
+        ├── tox.ini
+        ├── xsoar_config.json
+ 
+ ````
+ 
+5. **Merge the changes to the master repository.**
+   
+   The content is uploaded to the artifacts server or directly to the machine.
+ If using an artifact server, open the folder in the artifact server. The hierarchy should appear similar to this:
+ 
+ ````
+    ├── builds
+    │   ├── branch-name
+    │   │   ├── packs
+    │   │   │   ├── pack1
+    │   │   │   │   ├── 1.0.0
+    │   │   │   │   │   ├── pack1.zip
+    │   │   │   ├── pack2
+    │   │   │   │   ├── 1.0.1
+    │   │   │   │   │   ├── pack2.zip
+    │   │   │   ├── 
+    │   ├── 
+    ├── production
+    │   ├── packs
+    │   │   ├── pack1
+    │   │   │   ├── 1.0.0
+    │   │   │   │   │   ├── pack1.zip
+    │   │   │   ├── 1.0.1
+    │   │   │   │   │   ├── pack1.zip
+    │   │   │   ├── 1.1.0
+    │   │   │   │   │   ├── pack1.zip
+    │   │   │   ├── 
+    │   │   ├── pack2
+    │   │   │   ├── 1.0.0
+    │   │   │   │   │   ├── pack2.zip
+    │   │   │   ├── 1.0.1
+    │   │   │   │   │   ├── pack2.zip
+    │   │   │   ├── 1.0.2
+    │   │   │   │   │   ├── pack2.zip
+    │   │   │   ├── 
+    │   │   ├──
+
+ ````
+
+6. **(Non-artifact server) Install the content to Cortex XSOAR**.
+<br/> In Cortex XSOAR go to Marketplace and install the custom content packs. 
+Ensure that the server configuration (Settings>ABOUT>Troubleshooting)  `content.pack.verify` is set to `false` (enables you to import custom Content Packs that are not provided by Cortex XSOAR). 
```

Comment:
1. In the non-artifacts-server option after the upload the packs already appear in the Installed Packs in the marketplace.
2. We don't need to configure the key `content.pack.verify` - its append automatically when using the demisto SDK Upload command.